### DISCUSSION TITLE
NavBar/NavButton property disabled

### DISCUSSION THREAD

**Discussion by Sinan-SchwarzIT** [SchwarzIT/onyx#1837] [Ideas]
Hi folks,

currently, the NavButton does not have a property "disabled", which would show that there is a nav button, but it is greyed out and not clickable.

Use Cases:
- Display of upcoming features / new pages

Kind regards

**Comment by mj-hof** [comment]
Hi @Sinan-SchwarzIT 
Thank you for your input. We created a ticket out of it (#1862) and will start to work on this topic soon.
